### ISSUE TITLE
The date for a future backup is changed when you reload the page

### ISSUE THREAD

**Issue by Gamemap** [nextcloud/backup#142]
I created my first restore point and in the `restore point history ` it says it will be done in 5 minutes. When I reload the page (F5) the 5 minutes are counted from the point. You can repeat this again and again. If you wait the 5 minutes without reloading the page, nothing happens (no backup is created).

Maybe it is because I am trying to load the backup to an external hard drive. I did not manage to change the owner of the hard disk or a folder from root to e.g. www-data. Is it necessary to change the owner?

- Debian 11 (bullseye)
- Apache2
- Nextcloud 23

**Comment by mastermns** [comment]
I am not sure if this will solve the issue but yes  the php user needs to be owner of the backup folder and have sufficient rights.

**Comment by Gamemap** [comment]
I think I found the problem.

The background jobs were set to `AJAX` and needs to be set to `Cron` to work automatically.

@ArtificialOwl Maybe this could be added to the description on the apps page (https://apps.nextcloud.com/apps/backup). 
(I think the apps page on the server updates automaticly)

**Comment by ArtificialOwl** [comment]
yes, this should be written somewhere. the 5 minutes is just an estimation of the next tick of the Cron
